app/controllers/provider/admin/cms/templates_controller.rb

```diff
@@ -31,7 +31,18 @@ def update
     @page.build_version if params[:version]
 
     if @page.save
-      success_update
+      msg = publish_or_hide_page_with_message
```

Comment:
We can skip this variable assignment and just call `publish_or_hide_page_with_message` directly.